File: HW/HW2 - HW2 學生成績資料卡/Project1/Date.cpp
#include "Date.h"

Date::Date()
{
	month = 1;
	day = 1;
	year = 1;
}

void Date::setDate(int month, int day, int year)
{
	this->month = month;
	this->day = day;
	this-> year = year;
}

void Date::setYear(int year)
{
	this->year = year;
}

int Date::getYear()
{
	return year;
}

void Date::print()
{
	cout << month << "/" << day << "/" << year;
}

File: python/StudentCard_Composition/Date.py
class Date:

    def __init__(self, month, day, year):
        self.__month = month
        self.__day = day 
        self.__year = year

    def setMonth(self, month):
        self.__month = month 

    def setDay(self, day):
        self.__day = day

    def setYear(self, year):
        self.__year = year

    def getMonth(self):
        return self.__month  

    def getDay(self):
        return self.__day  

    def getYear(self):
        return self.__year  

    def toString(self):
        return str(self.getMonth()) + "\\" + str(self.getDay()) + "\\" + str(self.getYear())